ansible/roles/rrdstats/tasks/main.yml

```diff
@@ -0,0 +1,99 @@
+---
+- name: Install rrdtool
+  ansible.builtin.package:
+    name: rrdtool
+    state: present
+  tags:
+    - role::rrdstats
+
+- name: Template the statistics script
+  ansible.builtin.template:
+    src: generate-rrdtool-stats.sh.j2
+    dest: "{{ rrdstats_script_path }}"
+    owner: root
+    group: root
+    mode: "0555"
+  tags:
+    - role::rrdstats
+
+- name: Configure PostgreSQL user
```

Comment:
I think it makes sense to remove this and let the pg role handle it, since the pg role is already a dependency of this one.